.github/workflows/e2e-rancher-upgrade-fleet-to-head-ci.yml

```diff
@@ -23,88 +57,62 @@ env:
   SETUP_K3S_VERSION: 'v1.30.2-k3s2'
 
 jobs:
-  rancher-fleet-integration:
-    runs-on: ubuntu-latest
+  rancher-integration:
+    runs-on: runs-on,runner=4cpu-linux-x64,run-id=${{ github.run_id }}
 
     steps:
-      -
+      - name: Add paths into PATH
```

Comment:
I find the style with the empty newline after `-` very readable, as it pronounces the blocks. We also use it nearly all other workflows.